validation/src/shared_table/mod.rs

```diff
@@ -347,18 +370,30 @@ impl<Fetch, F, Err> Future for PrimedParachainWork<Fetch, F>
 				GenericStatement::Invalid(candidate_hash),
 				Validation::Invalid(pov_block),
 			),
-			Ok(outgoing_targeted) => {
-				let outgoing_queues = crate::outgoing_queues(&outgoing_targeted)
-					.map(|(_target, root, data)| (root, data))
-					.collect();
+			Ok((outgoing_targeted, our_chunk)) => {
+				let outgoing_queues = outgoing_targeted.clone().into();
 
-				self.inner.availability_store.make_available(Data {
+				let chunks = erasure::obtain_chunks(
+					self.inner.n_validators,
+					&pov_block.block_data,
+					Some(&outgoing_queues)).unwrap();
```

Comment:
clear out this and ALL OTHER `unwrap` calls. We have a strict policy on unwraps and I think I've pointed this out before.

Reply:
done